### ISSUE TITLE
tig status: Add "branch up-to-date with remote"

### ISSUE THREAD

**Issue by ffes** [jonas/tig#460]
When I use `git status` it shows if your current branch is up-to-date with the remote.

```
On branch master
Your branch is up-to-date with 'origin/master'.
```
or
```
On branch master
Your branch is ahead of 'origin/master' by 1 commit
```

`tig status` does not show this information. Simply adding it to the first line would be great:
```
On branch master. Your branch is up-to-date with 'origin/master'.
```

**Comment by netei** [comment]
I personnaly like this idea

**Comment by ffes** [comment]
For some reason this one was not automatically closed when the #504 was merged.
